pkg/storagepolicy/sdkstoragepolicy.go

```diff
@@ -0,0 +1,263 @@
+package storagepolicy
+
+import (
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/portworx/kvdb"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/libopenstorage/openstorage/api"
+	"github.com/libopenstorage/openstorage/pkg/jsonpb"
+)
+
+// SdkPolicyManager is an implementation of the
+// Storage Policy Manager for the SDK
+type SdkPolicyManager struct {
+	kv kvdb.Kvdb
+}
+
+const (
+	policyPrefix = "/storage/policy"
+	policyPath   = "/policies"
+	EnforcePath  = "/storage/policy/enforce"
+)
+
+var (
+	// Check interface
+	_ PolicyManager = &SdkPolicyManager{}
+
+	errPolicyManagerNotInitialized = errors.New("Policy Manager is not initialized")
```

Comment:
Why have these if they are not even exported? Just use fmt.Errorf() in the appropriate return.